packages/yaml/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@rollup/plugin-yaml",
+  "version": "2.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Convert YAML files to ES6 modules",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "author": "rollup",
+  "homepage": "https://github.com/rollup/plugins/packages/yaml/#readme",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "rollup -c",
+    "ci:coverage": "nyc pnpm run test && nyc report --reporter=text-lcov > coverage.lcov",
+    "ci:lint": "pnpm run build && pnpm run lint && pnpm run security",
+    "ci:lint:commits": "commitlint --from=${CIRCLE_BRANCH} --to=${CIRCLE_SHA1}",
+    "ci:test": "pnpm run test -- --verbose",
+    "lint": "pnpm run lint:js && pnpm run lint:docs && pnpm run lint:package",
+    "lint:docs": "prettier --single-quote --write README.md",
+    "lint:js": "eslint --fix --cache src test",
+    "lint:package": "prettier --write package.json --plugin=prettier-plugin-package",
+    "prebuild": "del-cli dist",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm run lint",
+    "pretest": "npm run build",
+    "security": "echo 'pnpm needs `npm audit` support'",
+    "test": "ava"
+  },
+  "files": [
+    "dist",
+    "README.md",
+    "LICENSE"
+  ],
+  "keywords": [
+    "rollup",
+    "plugin",
+    "yaml"
+  ],
+  "peerDependencies": {
+    "rollup": "^1.20.0"
+  },
+  "dependencies": {
+    "@rollup/plugin-json": "^4.0.0",
```

Comment:
Am I being dense or is `@rollup/plugin-json` not used by this package?

Reply:
this is interesting. I'll look into it